### PR TITLE
build: Latest go-mods, config and Dockerfile fix

### PR DESCRIPTION
**Pull request by ejlee3**
Signed-off-by: Elizabeth J Lee <elizabeth.j.lee@intel.com>

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/device-rest-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] I am not introducing a breaking change (if you are, flag in conventional commit message with `BREAKING CHANGE:` describing the break)
- [x] I am not introducing a new dependency (add notes below if you are)
- [ ] I have added unit tests for the new feature or bug fix (if not, why?) **N/A**
- [ ] I have fully tested (add details below) this the new feature or bug fix (if not, why?) **N/A**
- [ ] I have opened a PR for the related docs change (if not, why?)  **N/A**
  <link to docs PR>

### CODE REVIEW

File: cmd/res/configuration.toml [hunk 1]
```diff
@@ -11,6 +11,20 @@ LogLevel = "INFO"
...
+    PublishTopicPrefix  = "edgex/telemetry" # /<service-name>/<metric-name> will be added to this Publish Topic prefix
+      [Writable.Telemetry.Metrics] # All service's metric names must be present in this list.
+      # Device SDK provided Service Metrics
+      EventsSent = true
+      ReadingsSent = true
+      # Common Security Service Metrics
+      SecuritySecretsRequested = true
+      SecuritySecretsStored = true
+      SecurityConsulTokensRequested = true
+      SecurityConsulTokenDuration = true
```
Comment (lenny-intel) [inline cmd/res/configuration.toml:25]: These need to be set to false. Yep, I need to fix that on Device MQTT. ;-)

**Review by lenny-intel — CHANGES_REQUESTED** [review]

File: cmd/res/configuration.toml (hunk 1 above)
Comment (ejlee3) [reply cmd/res/configuration.toml]: done

**Review by lenny-intel — APPROVED** [review]
LGTM

**Review by lenny-intel — APPROVED** [review]
LGTM

**Pull request merged**